buildroot/rebuild.sh

```diff
@@ -21,15 +26,14 @@ done
 # Download latest buildroot release
 BUILDROOT_VERSION=2019.02.3
 if [[ -d src ]]; then
-    cd src || exit 1
-    if [[ $(git describe --exact-match --tags HEAD) != "${BUILDROOT_VERSION}" ]]; then
-        git fetch origin ${BUILDROOT_VERSION}
-        git checkout ${BUILDROOT_VERSION}
+    if [[ $(cd src && make print-version | cut -d - -f 1 >/dev/null) != "${BUILDROOT_VERSION}" ]]; then
```

Comment:
if you redirect stdout from `cut` to `/dev/null`, is the subshell ever going to return a string to compare against `"${BUILDROOT_VERSION}"`?

Reply:
Ah whoops, this should be `2>/dev/null` :(